doc/doc-site/docs/overview/orders.md

```diff
@@ -215,297 +215,214 @@ Econia does not use a critical height of 1. The actual critical height is define
 
-In addition to a base/quote trading pair, each market in Econia additionally contains the following parameters, which are selected by the market registrant during registration:
+Market participants should be able to express prices that are close enough together to be effectively continuous while being far enough apart to reflect meaningful movement with a single increment.
 
-| Parameter          | Meaning                     | Example     |
-| ------------------ | --------------------------- | ----------- |
-| Lot size           | Order size granularity      | 0.1 `APT`   |
-| Tick size          | Price granularity           | 0.01 `USDC` |
-| Minimum order size | Smallest allowed order size | 0.5 `APT`   |
+- **Too granular:** Almost every order occupies its own price level, and user interfaces struggle to display a useful segment of the order book before running out of visual real-estate. Maximum price may be too low (not common).
+- **Too coarse:** A single increment (or decrement) in price reflects a dramatic shift in value whose magnitude exceeds those of smaller movements that are still meaningful to market participants. Minimum price may be too high (common).
+- **Just right:** Prices are effectively continuous in that any meaningful movement is always expressible as some increment or decrement in price. Orders coalesce into useful price levels, and the minimum/maximum prices are not relevant concerns to market action.
 
-On this market, an order for 7.8 `APT` at a price of 5.23 `USDC` per `APT` would be valid, but the following would be invalid:
+#### Minimum order size
 
-| Size | Price | Reason             |
-| ---- | ----- | ------------------ |
-| 7.85 | 5.23  | Size too granular  |
-| 7.8  | 5.235 | Price too granular |
-| 0.4  | 5.23  | Size too small     |
+Market participants should be able to create a limit order as long as it's for a meaningful amount of asset.
+Each individual limit order affected by a market order adds to the network fee cost of the market order's execution.
+Thus, it's in every participant's interest that every limit order has a meaningful enough size to include in the execution of a market order.
 
-Econia's matching engine uses integers rather than decimals, such that conversion from decimal amounts to integer amounts requires the `aptos_framework::coin::CoinInfo.decimals` for a given `CoinType`:
+- **Too low:** The limit order books become polluted with small orders and it becomes difficult or impossible to execute a market order without paying a significant network fee. Denial-of-Service is a concern at this point because it becomes possible to "shut down" trading by spamming limit orders.
+- **Too high:** Some or many meaningful market participants are cut out of trading action and thus must take their liquidity somewhere else.
+- **Just right:** Nobody with good intentions is cut out of liquidity provision based on the amount of asset available to them, as long as their amount is meaningful.
```

Comment:
"Anyone who wants to provide liquidity can, even at smaller amounts, but malicious actors are denied an attack vector"